docs/composables/usekeystroke.md

```diff
@@ -0,0 +1,137 @@
+<!--
+ - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+ - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+Extended version of [VueUse onKeyStroke](https://vueuse.org/core/onKeyStroke/)
+
+This composable allows to use keyboard shortcuts in the application.
+It respects Nextcloud's value of ```OCP.Accessibility.disableKeyboardShortcuts``` parameter.
+
+### Usage
+```js static
+import { useKeystroke } from '@nextcloud/vue/dist/Composables/useKeyStroke/index.js'
+
+const stop = useKeystroke(keys, callback, modifiers)
+```
+where:
+- `keys`: string / array of strings representing the key or keys to listen to (`true` to listen to any key)
+
+	See [KeyboardEvent.key Value column](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values) for possible values
+- `callback`: a function to be called when the key is pressed. Before called, it will be checked whether keyboard shortcuts are disabled, or interactive element is currently focused, or whether modifiers should be applied
+- `modifiers`: modifiers to be applied to the shortcut:
+  - `target`: the HTML element (or Vue ref) to listen to the event (`document` by default)
+  - `push`: whether the event should be triggered on both keydown and keyup
+  - `press`: whether the event should be triggered, continuously generating events
+  - `ctrl`: whether the Ctrl key should be pressed
+  - `alt`: whether the Alt key should be pressed
+  - `shift`: whether the Shift key should be pressed
+  - `stop`: prevents propagation of the event in the capturing and bubbling phases
+  - `prevent`: prevents the default action of the event
```

Comment:
Although `target`, `press` are present in Vueuse, I'd not add support for it until its required to not increate API we want to support.